### PR TITLE
Save optimizer config

### PR DESCRIPTION
**Pull request by twolodzko**
Most of the changes are inherited from the second PR https://github.com/neptune-ai/neptune-tensorflow-keras/pull/14

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
   to keep the metadata better organized. `None` is not accepted anymore as a value of the argument.
 - We are additionally saving the model summary.
```
[the remark this answers is not in the document]
Comment (twolodzko) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
...
 - We fixed the dependencies for TensorFlow to version >= 2.0.0.
```
[the remark this answers is not in the document]
Comment (twolodzko) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,7 @@
...
+- We are saving the parameters of the optimizer used for training the model.
```
[the remark this answers is not in the document]
Comment (twolodzko) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,7 @@
   to keep the metadata better organized. `None` is not accepted anymore as a value of the argument.
```
[the remark this answers is not in the document]
Comment (twolodzko) [reply CHANGELOG.md]: Done

**Review by Raalsky — APPROVED** [review]

**Pull request merged**